### DISCUSSION TITLE
How do I enable totp on my cosmos instance?

### DISCUSSION THREAD

**Discussion by kun-codes** [azukaar/Cosmos-Server#333] [Q&A] [answered]
I can't find it in the docs. However I did find some commits related to that.

https://github.com/azukaar/Cosmos-Server/commit/3811e3131e9a3a3fc8053bfcbf035f9aa5312798
https://github.com/azukaar/Cosmos-Server/commit/714d56077464ca095dcac36246f714fa729b0674

Thanks

**Comment by azukaar** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/c6717713-287a-4ed7-b6bf-845db9cfb184)

**Comment by kun-codes** [comment]
Thanks it worked
